NavigateU/NavigateU/Coordinator/MainFlowCoordinator.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MainFlowCoordinator.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 29.05.2024.
+//
+
+import Foundation
+import UIKit
+
+protocol MainFlowCoordinatorProtocol: AnyObject {
+    func mainFlowSignOutUser()
+}
+
+class MainFlowCoordinator: Coordinator {
+
+    var navigationController: UINavigationController
+    private var mainFlowCoordinatorProtocol: MainFlowCoordinatorProtocol?
+
+    init(navigationController: UINavigationController, mainFlowCoordinatorProtocol: MainFlowCoordinatorProtocol) {
+        self.navigationController = navigationController
+        self.mainFlowCoordinatorProtocol = mainFlowCoordinatorProtocol
```

Comment:
looks like this property is useless

also it's look very strange, this protocol should be called sth like MainFlowCoordinatorOutput (because -protocol usually describes current entity, not another)